packages/suite/src/views/password-manager/PasswordManager/PasswordManager.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Button } from '@trezor/components';
+
+import { SectionItem, TextColumn, ActionColumn } from 'src/components/suite';
+import { useDevice, usePasswords } from 'src/hooks/suite';
+import { getNextId } from 'src/utils/suite/passwords';
+
+import { EntryForm } from './EntryForm';
+import { PasswordsList } from './PasswordsList';
+import { TagsList } from './TagsList';
+
+const PasswordManagerBody = styled.div`
+    display: flex;
+    flex: 1;
+    flex-direction: column;
+`;
+
+export const PasswordManager = () => {
+    const {
+        entries,
+        tags,
+        entriesByTag,
+        isSomeTagSelected,
+        config,
+        fileName,
+        selectedTags,
+        setSelectedTags,
+        connect,
+        disconnect,
+        selectedProvider,
+        providerConnecting,
+        savePasswords,
+        device,
+    } = usePasswords();
+
+    const { isLocked } = useDevice();
+    const isDeviceLocked = isLocked();
+
+    const [formActive, setFormActive] = useState<null | number>(null);
+
+    if (!device?.state) {
+        return <SectionItem>Connect and authorized device</SectionItem>;
+    }
+    if (providerConnecting) {
+        return <SectionItem>Connecting...</SectionItem>;
+    }
+
+    if (!selectedProvider || !fileName) {
+        return (
+            <SectionItem>
+                <TextColumn
+                    title="Trezor password manager"
+                    description="Re-implementation of former Trezor Password Manager webextension"
```

Comment:
Translations are not required under experimental features?

Reply:
Apparently they are not.

Reply:
nope, don't want to burden translators with something that might be thrown away